p3-frontend/angular.json

```diff
@@ -54,13 +54,13 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "500kb",
-                  "maximumError": "1mb"
+                  "maximumWarning": "1mb",
+                  "maximumError": "2mb"
                 },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "2kb",
-                  "maximumError": "4kb"
+                  "maximumWarning": "10kb",
+                  "maximumError": "20kb"
```

Comment:
a change like this needs explanation in the pr description.
if we're over one of these limits then that should probably be an item for the devs to look into; if it's blocking the build then we could raise the error threshold but i guess the warning threshold could stay the same

Reply:
replaced warning values to those from main branch.